src/material/core/theming/customize-tokens.scss

```diff
@@ -0,0 +1,35 @@
+@use 'sass:map';
+@use '../../core/style/sass-utils';
+@use '../../core/tokens/token-utils';
+@use '../../core/tokens/m2/mat/checkbox' as mat-checkbox-tokens;
+@use '../../core/tokens/m2/mdc/checkbox' as mdc-checkbox-tokens;
+
+@mixin customize-checkbox-tokens($tokens: ()) {
```

Comment:
I wonder if we can generalize this logic so we don't have to repeat it for each component.

Reply:
Yeah, I was going to do this in a follow-up PR. I create a new commit to demonstrate three components using a single general `customize-tokens` mixin